README.md

```diff
@@ -126,6 +126,8 @@ To specify how the actions transform the state tree, you write pure _reducers_.
 That's it!
 
 ```js
+// This assumes you are running in a Node environment. If you are running in a browser environment,
+// import from 'redux/es/redux.mjs' instead.
```

Comment:
Too avoid confusion and people copy-pasting "import from 'redux/es/redux.mjs'" it might be better to actually just write out the full line "import { createStore } from 'redux/es/redux.mjs'"